packages/icons/src/types.ts

```diff
@@ -7,6 +7,7 @@ import type {
 export interface IconProps extends SVGAttributes<SVGElement> {
   children?: never;
   color?: string;
+  size?: "small" | "big";
```

Comment:
proposal, instead using small/big, lets literally use 16/22 those are like names

Reply:
actually wait we don't even need the enum, just use a number and set it for both width and hight

Reply:
16 by default